### PR TITLE
chore: move to rum enhancer v2 #4

### LINKED ISSUE #4: Chore: Update to Rum Enhancer V2
### Description
Adobe is upgrading the Rum Tracker on the VG family of sites.

### Notes
Open pull request: https://github.com/aemsites/vg-partsasist/pull/3

### PR DESCRIPTION
**Pull request by shsteimer**
Fix #4
https://rum-v2--vg-partsasist--aemsites.aem.live/?rum=on

### CODE REVIEW

File: scripts/delayed.js
Comment (alexiscoelho) [inline scripts/delayed.js:5]: Why has this been removed?

File: scripts/scripts.js
Comment (alexiscoelho) [inline scripts/scripts.js]: Why has the above been removed and why the add of `sampleRUM.enhance();`

File: scripts/delayed.js
Comment (shsteimer) [reply scripts/delayed.js:5]: The call to `sampleRUM.enhance` (which you'll note in latest changes occurs in `updateSectionStatus`) loads what we call the rum enhancer, see https://github.com/adobe/helix-rum-enhancer

That takes care of all the collection calls removed in this PR, without customers having to update their codebases as we make rum collection changes. So this is a one time update to get you on rum v2 using the new rum enhancer, but as we add more features to what rum we collect, no future changes should be needed on your part

**Comment by alexiscoelho** [comment]
Can this be moved to be merged on develop or do you need it merged directly into main for some reason?
Also, I am waiting for a ticket/issue to assign to this before merge... Will come back to you asap

**Comment by dkuntze** [comment]
it can go wherever as long as it gets merged into main at some point

**Review by alexiscoelho — APPROVED** [review]

**Pull request merged**
